### PR TITLE
Jupyter packaging v0.10.4

### PR DESCRIPTION
**Pull request by ansthomas**
Requirements from https://github.com/jupyter/jupyter-packaging/blob/master/pyproject.toml
https://github.com/jupyter/jupyter-packaging/blob/master/setup.cfg

Actions

Added/updated dependencies
Added doc_url and test commands
Updated source url

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,15 +16,27 @@ build:
...
-    - python >=3.5
+    - python >=3.6
     - pip
+    - setuptools >=46.4.0
+    - wheel
+    - packaging
   run:
-    - python >=3.5
+    - python >=3.6
     - setuptools
```
Comment (antkneed) [inline recipe/meta.yaml]:
```suggestion
```

We should remove this if we are pinning below (`- setuptools >=46.4.0)`.

**Review by antkneed — CHANGES_REQUESTED** [review]
Only one thing to change. 👍

**Comment by skupr-anaconda** [comment]
Looks good. Only duplication of setuptools is in the run section

**Review by antkneed — APPROVED** [review]
👍

**Pull request merged**